src/libraries/StackUint256.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.4.13;
+
+import "ROOT/libraries/DelegationTarget.sol";
+import "ROOT/legacy_reputation/Ownable.sol";
+
+contract StackUint256 is DelegationTarget, Ownable {
+    
+    uint256[] public collection;
+    uint256 public head;
+    address public owner;
+    bool public initialized;
```

Comment:
Thinking about applying this policy to every contract, including the libraries, is quickly making me regret going along with the chosen policy... (though I recognize that the other options each had their own set of problems).